src/Youtube.js

```diff
@@ -61,6 +61,19 @@ THE SOFTWARE. */
     },
 
     dispose: function() {
+      if (this.ytPlayer) {
+        //Dispose of the YouTube Player
+        this.ytPlayer.stopVideo();
+        this.ytPlayer.destroy();
+      } else {
+        //YouTube API hasn't finished loading or the player is already disposed
+        var index = Youtube.apiReadyQueue.indexOf(this);
+        if (index !== -1) {
+          Youtube.apiReadyQueue.splice(index, 1);
+        }
+      }
+      this.ytPlayer = null;
+
```

Comment:
I wonder if we shouldn't call `Tech.prototype.dispose()` at some point in here...